package.json

```diff
@@ -23,7 +23,8 @@
   "author": "felixge+thlorenz, adapted by Mike McNeil",
   "license": "MIT",
   "dependencies": {
-    "lodash": "3.10.1"
+    "lodash": "3.10.1",
```

Comment:
Might as well take this to `@sailshq/lodash` while we're at it, since it'll consolidate the dependencies between this package and `merge-dictionaries` when this package is installed, but I'll follow up and do that afterwards-- that's just an unrelated optimization.